UI/window-basic-main.cpp

```diff
@@ -4847,6 +4847,20 @@ void OBSBasic::changeEvent(QEvent *event)
 			if (previewEnabled)
 				EnablePreviewDisplay(true);
 		}
+#ifdef __APPLE__
+	} else if (event->type() == QEvent::StyleChange) {
+		const char *theme_name = config_get_string(
+			App()->GlobalConfig(), "General", "CurrentTheme3");
+		if (theme_name && strcmp(theme_name, "Yami") == 0) {
```

Comment:
This could get messy when we add other color palettes of Yami (gray, light, etc).

Reply:
I could add a `font-size-[platform]` in `OBSThemeMeta {}` to do this programmatically rather than hardcoding themes in some way. It's the kind of thing I made `OBSThemeMeta` for.

Reply:
That could probably work as well, although you could run into the same issue with differing font sizes throughout the app.
(See the second comment:)
> This overrides every font-size in the app, which is an issue as not all font sizes are originally 10 (see the Yami.qss file for examples, like 10.5 in QDockWidget or 26 in the about dialog).